### DISCUSSION TITLE
Access Music Assistant (addon) outside of HASS

### DISCUSSION THREAD

**Discussion by dblinick** [music-assistant/support#1189] [Q&A] [answered]
Hi 
Is there a way to access the MASS UI from outside of HASS - I figure if it is rendering in the HASS iframe, there must be a way to just directly access, but am not seeing any documentation on how to do so?

Thanks,
-Dan

**Comment by OzGav** [comment]
Port 8095

**Reply by marcelveldt** [reply, marked as answer]
http://YOURIPHERE:8095 should do the trick (for now)

**Comment by dblinick** [comment]
Perfect - thank you - I will close!
